src/client/main.js

```diff
@@ -67,7 +67,12 @@ export function define(cell) {
 function noop() {}
 
 function clear(root) {
-  for (const v of root._nodes) v.remove();
+  delete root._inspectorState;
```

Comment:
Is this supposed to be `_inspectorExpanded`?

Reply:
oops, yes